src/App.tsx

```diff
@@ -17,18 +54,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                todos={todos}
+                setTodos={setVisibleTodos}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length ? (
```

Comment:
better to start with positive condition, it is easier to read
Also, the questionmark should be moved to a new line
```suggestion
              {!todos.length 
                ? (
```